plugins/dynamix/ArrayOperation.page

```diff
@@ -276,15 +276,15 @@ devices.on('message', function(data) {
     if ($('#'+table[0]).length) $('#'+table[0]).html(table[1]);
   }
   display_diskio();
+<?if ($var['fsState']=='Stopped'):?>
+  setTimeout(function(){devices.stop();},0);
```

Comment:
Just at tip, `setImmediate` would be a little quicker when using `0` as the time since `setTimeout` still needs to lookup the timer on run unlike `setImmediate`.